backend/game/cards/discard_cards/turn_manipulation_cards/frenzy.py

```diff
@@ -0,0 +1,10 @@
+from game.cards.discard_card import DiscardCard
+
+
+class Frenzy(DiscardCard):
+    def __init__(self):
+        super().__init__("Frenzy", 7,
+                         "Take another turn after this one", "")
```

Comment:
I had previously not been putting anything for footnote field if it was empty

Reply:
Good call.